docs/variants/msi_z690/community/community-hcl.md

```diff
@@ -0,0 +1,31 @@
+# Community HCL - MSI Z690-A WIFI DDR4
+
+## About
+
+The following document contains information about hardware compatible
+with the MSI Z690-A WIFI DDR4 platform with Dasharo firmware.
+
+The following list is created based on the information from the
+community - people which are currently using the platform with Dasharo
+firmware.
+
+This document contains three main sections:
+- `Testing method` in which we are describing our preferred testing method.
+- `Hardware Compatible List` in which in a labelled manner we show all
+    compatible hardware.
+- `Support us` - in which we describe the method of how it is possible to
+    contribute to the development of this list.
+
+## Testing method
+
+## Hardware Compatible List
+
+| No.  | Supported hardware                                | Link to description | Notes               |
+|:-----|:--------------------------------------------------|:-------------------:|:-------------------:|
+| 1.   |                                                   |                     |                     |
+
+## Support Us
+
+Hardware compatible with Dasharo firmware you might report by:
+- using `Dasharo Issues project`, which is available under this
+    [link](https://github.com/Dasharo/dasharo-issues/issues).
```

Comment:
Do we have template for reporting HCL?

Reply:
As a potential tester, I would be lost. I don't know what exactly I should provide to help.